### PR TITLE
feat: Host Reservations Management

### PR DESCRIPTION
**Pull request by duolok**
Host can now manipulate his reservations. 

SS:
image

### CODE REVIEW

**Comment by duolok** [comment]
Note: Code duplication should be addressed (rushed development process).

**Comment by duolok** [comment]
Should be safe to merge with develop.

File: app/src/main/res/menu/host_nav_menu.xml [hunk 1]
```diff
@@ -40,14 +40,15 @@
...
+  
     <item
         android:id="@+id/nav_notifications"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:icon="@drawable/baseline_notifications_24"
         android:title="@string/notifications" />
     <item
-        android:id="@+id/nav_reports"
+        android:id="@+id/menu_reports"
```
Comment (vradovic) [inline app/src/main/res/menu/host_nav_menu.xml]: This breaks the navigation, it should stay nav_reports.

**Review by vradovic — APPROVED** [review]
Looks good but the id in host_nav_menu.xml should not be changed. I will commit the fix and merge this PR.

**Pull request merged**